File: startup-bench/exec/exec.js
const child_process = require("child_process");

const p = child_process.spawn("bash", ["-c", "echo Hi"]);

const output = [];
p.stdout.on("data", (data) => output.push(data));
p.on("close", (code) => {
  process.stdout.write(`${code} ${Buffer.concat(output)}`);
});

File: startup-bench/exec/exec.rs
use std::process::Command;

fn main() {
    let child = Command::new("bash")
        .arg("-c")
        .arg("echo Hi")
        .output()
        .unwrap_or_else(|e| panic!("failed to execute child: {}", e));

    print!("{} {}", child.status.code().unwrap(), String::from_utf8_lossy(&child.stdout));
}
